contracts/Governor/GovernorFactory.sol

```diff
@@ -95,19 +94,4 @@ contract GovernorFactory is ERC165, ModuleFactoryBase {
             abi.decode(data[1], (address))
         );
     }
-
-    /// @notice Returns whether a given interface ID is supported
-    /// @param interfaceId An interface ID bytes4 as defined by ERC-165
-    /// @return bool Indicates whether the interface is supported
-    function supportsInterface(bytes4 interfaceId)
-        public
-        view
-        virtual
-        override(ERC165, ModuleFactoryBase)
-        returns (bool)
-    {
-        return
-            interfaceId == type(IModuleFactory).interfaceId ||
```

Comment:
Ah, nice, this was just comparing against IModuleFactory in the first place, so we didn't lose anything by just removing it.